core/query/index.go

```diff
@@ -39,6 +39,7 @@ func (ind *Indexer) ProcessBlocks(ctx context.Context) {
 // saves all annotated transactions to the database.
 func (ind *Indexer) IndexTransactions(ctx context.Context, b *bc.Block) error {
 	<-ind.pinStore.PinWaiter("asset", b.Height)
+	<-ind.pinStore.PinWaiter("account", b.Height)
```

Comment:
Just double checking, this can't use account.PinName because it would be a dependency cycle?